Tests/ClientRuntimeTests/NetworkingTests/URLSession/FoundationStreamBridgeTests.swift

```diff
@@ -52,15 +47,21 @@ class FoundationStreamBridgeTests: XCTestCase {
             // Run a test for every possible data size up to the maximum
             let dataSize = min(run, maxDataSize)
 
-            // The buffer may be as small as 1 byte, up to 2x as big as the data size capped by maxBufferSize
-            let bufferSize = Int.random(in: 1...min(2 * dataSize, maxBufferSize))
+            // Sizes of the following buffers within the `FoundationStreamBridge` under test are randomized to
+            // try and elicit errors in the streaming process.
+
+            // The bridge buffer may be as small as 1 byte, up to 2x as big as the data size
+            let bridgeBufferSize = Int.random(in: 1...(2 * dataSize))
+
+            // The bound stream buffer may be as small as 1 byte, up to 2x as big as the data size
+            let boundStreamBufferSize = Int.random(in: 1...(2 * dataSize))
```

Comment:
Buffer sizes for the bridge buffer and the bound stream buffer are set separately.  (Before, these two buffers were set to the same size.)

This helps to create scenarios where either buffer may be partially consumed during streaming.